src/css/modal.css

```diff
@@ -2,6 +2,9 @@
   max-width: 768px;
   width: 100%;
   margin: 1rem auto;
+  display: none;
```

Comment:
The display: none property is not needed to hide the popup, I kindly suggest you to remove it.